src/pybind/mgr/cephadm/module.py

```diff
@@ -2359,7 +2359,23 @@ def add_mgr(self, spec):
         return self._add_daemon('mgr', spec, self._create_mgr)
 
     def _apply(self, spec):
-        self.log.info('Saving service %s spec' % spec.service_name())
+        if spec.placement.is_empty():
+            # fill in default placement
+            defaults = {
+                'mon': orchestrator.PlacementSpec(count=5),
```

Comment:
As long as this works, if we have two or 4 hosts registered!

Reply:
Yep, it works.

Right now, though, it silently degrades.  I wonder if in the future we want to have some health warning in these cases?